lib/storage/leveldb.go

```diff
@@ -402,6 +405,10 @@ func (st *LevelDBBackend) Walk(prefix string, option *WalkOption, walkFunc WalkF
 
 	var cnt uint64 = 0
 
+	if option.SkipCursor == true {
+		iter.Seek(st.makeKey(cursor))
```

Comment:
If `SkipCursor` is `true`, the item of cursor should be skipped, right?  If `SkipCursor` is `false`, the item of cursor will not be included to `walkFunc`. Before calling `iter.Next()`(`iterFunc`), this should be call `iter.Key()` and `iter.Value()` for cursor item.

Reply:
If `SkipCursor` is true,  the item is cursor skip by `iter.Seek`. 
if `SkipCursor` is false, the item is cursor is included in iterator.

Reply:
I think `iter.Seek()` does not skip the cursor item. Please correct me :)

Reply:
Oh, Thanks. @spikeekips :-)